### PR TITLE
Use scala-seed as the default

### PR DESCRIPTION
**Pull request by wsargent**
if it says scala-seed, it should say scala-seed

### CODE REVIEW

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-name=My Something Project
+name=scala-seed
```
Comment (eed3si9n) [inline src/main/g8/default.properties]: What's the motivation here?

I don't know if "My Something Project" is the best wording, would I think it's clear that you're suppose to replace it with the name of the project you're about to start.

**Review by eed3si9n — CHANGES_REQUESTED** [review]

File: src/main/g8/default.properties (hunk 1 above)
Comment (wsargent) [inline src/main/g8/default.properties]: Would 

```
name=Please enter name of the project you're about to start.
```

be better?

Comment (eed3si9n) [inline src/main/g8/default.properties]: It currently renders like this:

```
 sbt new scala/scala-seed.g8
...
[info] Set current project to work (in build file:/xxx/work/)

Minimum Scala build.

name [My Something Project]:
```

I don't know if it should be too much longer than that. `Hello World Project`?

Comment (wsargent) [inline src/main/g8/default.properties]: `Scala Seed Project`

Comment (eed3si9n) [inline src/main/g8/default.properties]: ok. let's go with that.

**Pull request merged**